editor/src/components/canvas/canvas-strategies/relative-move-strategy.tsx

```diff
@@ -0,0 +1,188 @@
+import { ElementPath } from 'src/core/shared/project-file-types'
+import { getLayoutProperty } from '../../../core/layout/getLayoutProperty'
+import { MetadataUtils, PropsOrJSXAttributes } from '../../../core/model/element-metadata-utils'
+import { isRight, right } from '../../../core/shared/either'
+import { ElementInstanceMetadata, isJSXElement } from '../../../core/shared/element-template'
+import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
+import { CanvasCommand } from '../commands/commands'
+import { setCssLengthProperty } from '../commands/set-css-length-command'
+import {
+  CanvasStrategy,
+  emptyStrategyApplicationResult,
+  getTargetPathsFromInteractionTarget,
+  strategyApplicationResult,
+} from './canvas-strategy-types'
+import { getDragTargets } from './shared-absolute-move-strategy-helpers'
+
+export const relativeMoveStrategy: CanvasStrategy = {
+  id: 'RELATIVE_MOVE',
+
+  name: () => 'Move (Relative)',
+
+  isApplicable: (canvasState, _interactionState, metadata) => {
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return false
+    }
+
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const meta = MetadataUtils.findElementByElementPath(metadata, last)
+    if (!meta) {
+      return false
+    }
+
+    // should we also support absolute elements, for which we'll do something like `CONVERT_TO_RELATIVE`?
+    return meta.specialSizeMeasurements.position === 'relative'
+  },
+
+  controlsToRender: [],
+
+  fitness: (canvasState, interactionState, _sessionState) => {
+    const { interactionData, activeControl } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag != null)) {
+      return 0
+    }
+    if (activeControl.type !== 'BOUNDING_AREA') {
+      return 0
+    }
+
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return 0
+    }
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const meta = MetadataUtils.findElementByElementPath(interactionState.latestMetadata, last)
```

Comment:
Minor, but can we stick with `metadata` for this please, as we use that (rather than `meta`) everywhere

Reply:
sure, thanks for pointing that out - it makes it a lot easier for me to pick up the small nit conventions 🙏